src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_FlowAnalysis.cs

```diff
@@ -307,7 +307,16 @@ static void G(C? x, Unknown y)
                 Diagnostic(ErrorCode.ERR_SingleTypeNameNotFound, "Unknown").WithArguments("Unknown").WithLocation(3, 24),
                 // (6,9): warning CS8602: Possible dereference of a null reference.
                 //         a[0].ToString();
-                Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "a[0]").WithLocation(6, 9));
+                Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "a[0]").WithLocation(6, 9),
+                // (8,9): warning CS8602: Possible dereference of a null reference.
+                //         b[0].ToString();
```

Comment:
> b[0].ToString(); [](start = 26, length = 17)

Please add comments like `// warn N` so reviewers don't have to count lines. Thanks

Reply:
Updated this test and will include differences between lines in future tests.

---
In reply to: [189330845](https://github.com/dotnet/roslyn/pull/26356#discussion_r189330845) [](ancestors = 189330845)